app/main.py

```diff
@@ -2,0 +1,26 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    route = command[2].split("/")
+    if command[0] != "mv":
+        return
+    if len(route) > 1:
+        route.pop()
+    else:
+        new_file_name = route[0]
+        with (
+            open(command[1], "r") as file_in,
+            open(new_file_name, "x") as file_out,
```

Comment:
Code Style: Variable names should be descriptive. 'file_in' and 'file_out' are not very descriptive. Consider renaming them to something more descriptive, like 'source_file' and 'destination_file'.